{"name": "  / 2", "status": "passed", "description": "\n        单元测试加法运算，预期异常\n        :param div_fail_data:加法的参数化异常数据\n        :param setup_class: 实例化计算器代码\n        :return: 断言结果\n        ", "steps": [{"name": "执行计算   / 2", "status": "passed", "start": 1624373024571, "stop": 1624373024571}], "attachments": [{"name": "log", "source": "7ac394e0-8198-4bf6-ba08-1e5eec2454a8-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "div_fail_data", "value": "[' ', 2, 'Exception']"}], "start": 1624373024571, "stop": 1624373024571, "uuid": "14a88d47-e4f5-4dfb-8f50-b1a060b2c83e", "historyId": "2dc5f5bc222a27ab6434eecb20cc8389", "testCaseId": "29827050ec27802c871b66c512fd6c0b", "fullName": "testing.test_calculator.TestCalculator#test_div_exception", "labels": [{"name": "feature", "value": "计算器测试用例"}, {"name": "story", "value": "异常数据的除法"}, {"name": "parentSuite", "value": "testing"}, {"name": "suite", "value": "test_calculator"}, {"name": "subSuite", "value": "TestCalculator"}, {"name": "host", "value": "MS-DPHROIDQRVFN"}, {"name": "thread", "value": "10556-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testing.test_calculator"}]}